src/site/content/en/learn/css/lists/index.md

```diff
@@ -0,0 +1,244 @@
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'jOwWGEy',
+  tab: 'css,result'
+} %}
+
+### `list-style` shorthand
+
+Now that we have all of these individual properties, we can use the [`list-style`](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style) shorthand to set all of our list styles in one line:
+
+```css
+list-style: <'list-style-type> || <'list-style-position> || <'list-style-image>
+```
+
+`list-style` allows you to declare combinations of one, two, or three of the `list-style` properties in any order. If `list-style-type` and `list-style-image` are both set, then` list-style-type` is used as a fallback if the image is unavailable.
+
+```css
+/* type */
+list-style: square;
+
+/* image */
+list-style: url('../img/shape.png');
+
+/* position */
+list-style: inside;
+
+/* type | position */
+list-style: georgian inside;
+
+/* type | image | position */
+list-style: lower-roman url('../img/shape.png') outside;
+
+/* Keyword value */
+list-style: none;
+
+/* Global values */
+list-style: inherit;
+list-style: initial;
+list-style: revert;
+list-style: unset;
+```
+
+This is the most commonly used property of the list styles covered in this section. One common application is `list-style: none` to hide default styles. Default styles come from the browser, and you often see reset stylesheets removing list styles like padding and margins. You can also use this shorthand to set styles, like `list-style: square inside;`
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'eYRJGme',
+  tab: 'result'
+} %}
+
+So far, the examples have focused on styling an entire list and list items, but what about a more granular approach?
+
+## ::marker pseudo
+
+The `list-item` marker element is the bullet, hyphen or roman numeral that helps indicate each list-item in your list.
+
+{% Img src="image/dQwiPAoChjTYzXYCNg6U6Tsnr3x1/pOWTl4BYyJI7qgbMVeYu.jpg", alt="A list with three items which shows that each of the bullets are ::marker pseudo elements", width="800", height="395" %}
+
+If you inspect the list in DevTools, you can see a `::marker` element for each of the list items, despite not declaring any in HTML. If you inspect the `::marker` further, you'll see the browser default styling for it.
+
+```css
+::marker {
+    unicode-bidi: isolate;
+    font-variant-numeric: tabular-nums;
+    text-transform: none;
+    text-indent: 0px !important;
+    text-align: start !important;
+    text-align-last: start !important;
+}
+```
+
+When you declare a list, each item is given a marker, despite there being no bullet point or roman numeral in your HTML. This is  a pseudo-element because the browser generates it for you, and provides a limited styling API to target it. [Learn more about the anatomy of the CSS bullet.](https://web.dev/css-marker-pseudo-element/) `::marker` currently has [limited support](https://developer.mozilla.org/en-US/docs/Web/CSS/::marker#browser_compatibility) in Safari.
```

Comment:
```suggestion
When you declare a list, each item is given a marker, despite there being no bullet point or roman numeral in your HTML. This is  a pseudo-element because the browser generates it for you, and provides a limited styling API to target it. [Learn more about the anatomy of the CSS bullet.](/css-marker-pseudo-element/) `::marker` currently has [limited support](https://developer.mozilla.org/en-US/docs/Web/CSS/::marker#browser_compatibility) in Safari.
```